File: code-coaches/easy/guard-flamingos/GuardFlamingos.swift
func main() {
    if let input = readLine(), let yardLength = Int(input),
        let input2 = readLine(), let yardWidth = Int(input2)
    {
        let flamingosToPurchase = yardLength + yardWidth
        print(flamingosToPurchase)
    }
}

main()

File: code-coaches/easy/guard-flamingos/GuardFlamingos.cs
using System;

public class GuardFlamingos
{
    static void Main(string[] args)
    {
        int yardLength = int.Parse(Console.ReadLine());
        int yardWidth = int.Parse(Console.ReadLine());
        int flamingosToPurchase = yardLength + yardWidth;
        Console.WriteLine(flamingosToPurchase);
    }
}